src/modules/polkadotSlotAuction/components/SupportProject/SupportProjectStyles.tsx

```diff
@@ -2,14 +2,14 @@ import { makeStyles } from '@material-ui/core/styles';
 
 export const useSupportProjectStyles = makeStyles(theme => ({
   container: {
-    background: '#000000',
+    background: '#0F0F0F',
     border: '1px solid #3F3F3F',
     boxShadow: '0px 4px 4px rgba(0, 0, 0, 0.25)',
     borderRadius: 65,
     padding: '52px 160px',
     position: 'relative',
     maxWidth: 1130,
-    margin: '0 auto',
+    margin: '60px auto',
```

Comment:
use theme.spacing() with any css numbers

```suggestion
    margin: theme.spacing(60, 'auto', 0)
```

you don't need bottom here